src/sentry/receivers/sentry_apps.py

```diff
@@ -75,6 +86,18 @@ def send_comment_webhooks(organization, issue, user, event, data=None):
             user_id=(user.id if user else None),
             data=data,
         )
+    if features.has("organizations:sentry-functions", organization, actor=user):
+        if user:
+            try:
+                data["user"] = serialize(User.objects.get(id=user.id), user, UserSerializer())
+            except User.DoesNotExist:
+                pass
+        try:
+            data["comment"] = serialize(Group.objects.get(id=issue.id))
```

Comment:
Serializing the group pulls in several other resources like stats and activity etc. Do you need all of that in the function calls?

Another thing to think about/consider is should the payloads we send to sentry-functions be similar to the payloads that we send to SentryApplication webhooks?

You have the key called `comment` but it contains the entire issue. Is that what you want, as it is different that the workflow hook below.